src/util/kaldi-table-inl.h

```diff
@@ -38,6 +40,25 @@ namespace kaldi {
 /// \addtogroup table_impl_types
 /// @{
 
+// In SequentialTableReaderScriptImpl and RandomAccessTableReaderScriptImple,
+// We use this function to separate the input string "line"
+// into the range specifier (e.g. [1:2,2:10]) and the data_rxfilename
```

Comment:
in this comment please give an example of the string 'line' itself, e.g. "1.ark:100[1:2,2:10]", and maybe other examples of inputs and outputs would be nice, e.g. without ranges or with pipes.
Although old code doesn't do this, best to use single quotes to surround variable names.